.github/workflows/build.yml

```diff
@@ -112,13 +112,31 @@ jobs:
           dune-cache: true
           opam-pin: false
 
+      - name: Pin dune
+        run: |
+          opam pin add dune.3.17 https://github.com/ocaml-wasm/dune.git#wasm_of_ocaml
+
       - run: opam install conf-pkg-config
         if: runner.os == 'Windows'
 
       - run: opam install . --best-effort
         if: ${{ matrix.skip-test }}
 
-      - run: opam install . --with-test
+      - run: cat VERSION | xargs opam pin wasm_of_ocaml-compiler . -n --with-version 
```

Comment:
Do we really need to ping wasm_of_ocaml explicitly ?

Reply:
The issue is that the other packages take their version from the opam repository while `wasm_of_ocaml-compiler`, which is not in the repository, is given version `dev` by default.